sway-lib-std/src/bytes.sw

```diff
@@ -913,3 +955,29 @@ fn test_eq() {
     other.swap(0, 1);
     assert(bytes != other);
 }
+
+#[test()]
+fn test_sha256() {
+    use ::hash::sha256;
+    let (mut bytes, a, b, c) = setup();
+    bytes.push(0u8);
+    bytes.push(0u8);
+    bytes.push(0u8);
+    bytes.push(0u8);
+    bytes.push(0u8);
+
+    assert(sha256(362268190631264256) == bytes.sha256());
```

Comment:
Overall this looks good. Can you comment on how you derived this value `362268190631264256` ?